updated_main.py

```diff
@@ -3,57 +3,86 @@
 import bs4
 import time
 import json
+import os
+from email.message import EmailMessage
+import ssl
+import smtplib
 
 # Running a for loop to get the latest price of the GBP
 for i in range(100):
```

Comment:
Why 100?

Reply:
while loop bhi chala sakte par for testing person for loop use kiya tha

Reply:
Yes while loop is better. Lets use that.